test/e2e/egress_test.go

```diff
@@ -167,6 +167,8 @@ ip netns exec %[1]s /agnhost netexec
 	defer data.crdClient.CrdV1alpha2().Egresses().Delete(context.TODO(), egress.Name, metav1.DeleteOptions{})
 	assertClientIP(localPod, egressNodeIP)
 	assertClientIP(remotePod, egressNodeIP)
+	egressState, _ := data.crdClient.CrdV1alpha2().Egresses().Get(context.TODO(), egress.Name, metav1.GetOptions{})
+	assert.Equal(t, egressNode, egressState.Status.EgressNode, "Egress status not match")
```

Comment:
Would it cause the test flaky? The status update is asynchrounous, the client IP check is quite quick. Normally the test uses wait.Poll or wait.PollImmediate.

Reply:
done

Reply:
You should remove this assertion? otherwise it will still fail the test